test/IRGen/package_resilience.swift

```diff
@@ -1,20 +1,18 @@
 //
 // Unlike its counterparts in the other *_resilience.swift files, the goal is
 // for the package's component modules to all be considered within the same
-// resilience domain. This file ensures that we use direct access as much as
-// possible.
+// resilience domain. This file ensures that we use direct access to package
+// decls at use site as much as possible with -package-bypass-resilience-optimization.
```

Comment:
Thanks! While you're here, could you patch up my typo from the original PR introducing this test? https://github.com/apple/swift/blob/576e60638c43d7cfc1c1544e5ecce8b644c27edb/test/IRGen/package_resilience.swift#L224